tools/test_set_firewall_ns.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env python3
+
+import sys
+import json
+from cloudcix_primitives import set_firewall_ns
+
+cmd = sys.argv[1] if len(sys.argv) > 1 else None
+set_name = "IE_41"
+namespace_name = "ns1100"
+config_file = "/etc/cloudcix/pod/configs/config.json"
+
+if len(sys.argv) > 2:
+    namespace_name = sys.argv[2]
+if len(sys.argv) > 3:
+    set_name = sys.argv[3]
+
+status = None
+msg = None
+data = None
+
+if cmd == 'build':
+    status, msg = set_firewall_ns.build(namespace_name, set_name, 'ipv4_addr', config_file)
+elif cmd == 'scrub':
+    status, msg = set_firewall_ns.scrub(namespace_name, set_name, config_file)
+elif cmd == 'update': 
+    status, msg = set_firewall_ns.update(namespace_name, set_name, '8.8.8.8, 1.1.1.1', config_file)
```

Comment:
Please don't hardwire the set's content - make it configurable. Also, do we really want `elements` to be a flat string? Your code is up to spec, but I'm not sure I like this part of the spec...

Reply:
Clarification on this one: I'm fine with merging this with `elements` being a flat string for now - let's discuss the relative merits of flat string vs. list next week and create a follow up pull request as needed. Do make it a command line parameter, though.